openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/stf/SharedClasses.java

```diff
@@ -118,7 +118,7 @@ private Tests(String mnemonic, Class<?> testClass, String classArgs, Boolean use
         SCM20("-Xshareclasses:name=${cacheName},cacheDir=${cacheDir}${cacheOperation} -Xitn5000 -Xitsn0"), // default value for itn was 2000 at Nov 2010
         SCM21("-Xshareclasses:name=${cacheName},cacheDir=${cacheDir}${cacheOperation},nojitdata"), 
         SCM22("-Xshareclasses:name=${cacheName},cacheDir=${cacheDir}${cacheOperation} -Xscdmx18m"),
-        SCM23("-Xshareclasses:name=${cacheName},cacheDir=${cacheDir}${cacheOperation} -Xaot:forceAoT,count=1"), 
+        SCM23("-Xshareclasses:name=${cacheName},cacheDir=${cacheDir}${cacheOperation} -Xaot:forceAoT,count=1 -Xjit:{*Test_Init_*11.stringOperations(III)V|*Test_Init_*111.makeString(I)*}(traceInlining,traceOptTrees,log=inlining.log)"),
```

Comment:
This temporary change needs to be removed.

Reply:
Done.